.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,33 +1,26 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: "[BUG]"
+title: "A concise title here!"
 labels: bug
-assignees: tbaumer22
-
 ---
 
 **Describe the bug**
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Steps to reproduce the behavior.
+Steps to reproduce the behaviour.
```

Comment:
Did you really just go through and change all of the American spelling?

Reply:
You damn well bet I did.

Reply:
The only way this will ever be resolved is if one of us manages to change enough of it to the point where the other does not notice and/or cares.